### ISSUE TITLE
Most recent numpy version failing tests

### ISSUE THREAD

**Issue by JMGilbert** [ClimateImpactLab/dscim#62]
In the most recent version of numpy, we need to replace `**` with `np.float_power`
```
import xarray as xr
import numpy as np
dims = ["dim1", "dim2"]
array = xr.DataArray(
    data=np.array([[5.0, 10.0], [3.0, 2.0]]),
    dims=dims,
    coords=np.array([[1, 2], [3, 4]]),
)
weights = xr.DataArray(
    data=np.array([[0.4, 0.2], [0.1, 0.3]]),
    dims=dims,
    coords=np.array([[1, 2], [3, 4]]),
)  # sum of weights equals 1
eta = 10
exp = 1 / (1 - eta)
utility = np.true_divide(array ** (1 - eta), (1 - eta))
exp_utility = utility.weighted(weights).mean(dim=dims)
ce_array = (exp_utility * (1 - eta)) ** exp
print(ce_array.values == ((np.divide(array ** (1 - 10), 1 - 10) * weights).values.sum() * (1 - 10)) ** (1 / (1 - 10)))
```
prints `False`

while

```
import xarray as xr
import numpy as np
dims = ["dim1", "dim2"]
array = xr.DataArray(
    data=np.array([[5.0, 10.0], [3.0, 2.0]]),
    dims=dims,
    coords=np.array([[1, 2], [3, 4]]),
)
weights = xr.DataArray(
    data=np.array([[0.4, 0.2], [0.1, 0.3]]),
    dims=dims,
    coords=np.array([[1, 2], [3, 4]]),
)  # sum of weights equals 1
eta = 10
exp = 1 / (1 - eta)
utility = np.true_divide(np.float_power(array, (1 - eta)), (1 - eta))
exp_utility = utility.weighted(weights).mean(dim=dims)
ce_array = np.float_power((exp_utility * (1 - eta)), exp)
print(ce_array.values == np.float_power(((np.divide(np.float_power(array, (1 - 10)), 1 - 10) * weights).values.sum() * (1 - 10)),(1 / (1 - 10))))
```
returns `True` (the desired result)

**Comment by JMGilbert** [comment]
#63 fixes the `np.float_power` issue.

**Comment by JMGilbert** [comment]
The risk aversion gwr damage function tests still fail for numpy 1.24.1 -- may have to do with https://github.com/numpy/numpy/issues/22956

**Comment by kemccusker** [comment]
`dscim`'s dependency is pinned to `numpy==1.23.3` for now

**Comment by brews** [comment]
Yeah, I could get it to `numpy==1.24.0` but beyond that it starts to fail. Afraid I can't decipher what's going on in testing. Unfortunately, the logs to the failed tests in #63 have expired.

Normally, it can be helpful to check for deprecation warnings in local tests or CI run logs in the section after CI shows test results, but before test coverage gets uploaded. This usually calls out behavior that will break in future releases. We have warnings but it's unclear to me if they are directly related.

Also, looking at numpy's [changelog](https://numpy.org/doc/stable/release/1.24.1-notes.html) it's kinda weird that tests fail on v1.24.1 and not v1.24.0. Numpy v1.24.0 should have all the breaking behavior changes.

**Comment by brews** [comment]
A reminder that #11 might be tangled into this.

**Comment by brews** [comment]
Wat.

All the tests are now passing with numpy v1.25.2 in PR #119.

**Comment by JMGilbert** [comment]
So strange... I can't think of any reasons not to merge though!

**Comment by brews** [comment]
Cool. Thanks for the input. Merging away.

**Comment by brews** [comment]
Closed with PR #119
